utils/src/ooinstall/variants.py

```diff
@@ -49,9 +49,16 @@ def latest_version(self):
     ]
 )
 
-# Ordered list of variants we can install, first is the default.
-SUPPORTED_VARIANTS = (OSE, AEP)
+origin = Variant('origin', 'OpenShift Origin',
+    [
+        Version('1.2', 'origin'),
+        Version('1.1', 'origin')
```

Comment:
@tdawson ah, I had forgotten about that. 

@smunilla I would prefer that we avoid 1.1 here, since the documentation isn't going to mention it and I would like to avoid people installing and asking for support for 1.1.6 :)